style.css

```diff
@@ -75,6 +76,10 @@ h1 {
   margin: 2rem 0 0 4rem;
 }
 
+.up-menu a {
+  color: #fff;
+}
```

Comment:
- [ ] Your menu is working perfectly, just one thing is you have to make your desktop version match the figma design. Kindly make the navigations items appear on the desktop version, you have to change the `color: #fff;` to make them appear.

Your design
![image](https://user-images.githubusercontent.com/35535130/161746466-f1904dca-b9ee-4c24-ba48-321435d7741f.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/161746700-df308015-bbca-4c65-b470-c60fba5b5bb7.png)